database/challenge.go

```diff
@@ -184,15 +198,25 @@ func (db *DB) DeleteUnconfirmedUserUpdate(ctx context.Context, chID primitive.Ob
 
 // LoadFromRequest loads a ChallengeResponse by extracting its string form from
 // http.Request.
-func (cr *ChallengeResponse) LoadFromRequest(req *http.Request) error {
-	resp, err := hex.DecodeString(req.PostFormValue("response"))
+func (cr *ChallengeResponse) LoadFromRequest(body io.Reader) error {
```

Comment:
I think if you pass in the body we should also update the name of the method.

Reply:
Name updated.